packages/edit-site/src/components/editor/index.js

```diff
@@ -198,40 +185,12 @@ function Editor( { initialSettings } ) {
 											labels={ interfaceLabels }
 											drawer={ <NavigationSidebar /> }
 											secondarySidebar={
-												isInserterOpen ? (
-													<div
-														ref={
-															inserterDialogRef
-														}
-														{ ...inserterDialogProps }
-														className="edit-site-editor__inserter-panel"
-													>
-														<div className="edit-site-editor__inserter-panel-header">
-															<Button
-																icon={ close }
-																onClick={ () =>
-																	setIsInserterOpened(
-																		false
-																	)
-																}
-															/>
-														</div>
-														<div className="edit-site-editor__inserter-panel-content">
-															<Library
-																showInserterHelpPanel
-																onSelect={ () => {
-																	if (
-																		isMobile
-																	) {
-																		setIsInserterOpened(
-																			false
-																		);
-																	}
-																} }
-															/>
-														</div>
-													</div>
-												) : null
```

Comment:
Removing this `null` return value is what's causing the secondary sidebar from the interface skeleton to block the editor in mobile views (mentioned it [here](https://github.com/WordPress/gutenberg/pull/28637#issuecomment-785419490))